templates/parts/site-header.php

```diff
@@ -105,3 +105,24 @@ function add_css() {
 	// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 	echo $inline_css;
 }
+
+function add_social_tags( $html_title, $url, $html_description, $image_url ) {
+		echo '<meta name="twitter:card" content="summary" />' . "\n";
+		echo '<meta name="twitter:site" content="@WordPress" />' . "\n";
+		echo '<meta name="twitter:title" content="' . esc_attr( $html_title ) . '" />' . "\n";
+		echo '<meta name="twitter:description" content="' . esc_attr( $html_description ) . '" />' . "\n";
+		echo '<meta name="twitter:creator" content="@WordPress" />' . "\n";
+		echo '<meta name="twitter:image" content="' . esc_url( $image_url ) . '" />' . "\n";
+		echo '<meta name="twitter:image:alt" content="' . esc_attr( $html_title ) . '" />' . "\n";
+
+		echo '<meta property="og:url" content="' . esc_url( $url ) . '" />' . "\n";
+		echo '<meta property="og:title" content="' . esc_attr( $html_title ) . '" />' . "\n";
+		echo '<meta property="og:description" content="' . esc_attr( $html_description ) . '" />' . "\n";
+		echo '<meta property="og:site_name" content="' . esc_attr( get_bloginfo() ) . '" />' . "\n";
+		echo '<meta property="og:image:url" content="' . esc_url( $image_url ) . '" />' . "\n";
+		echo '<meta property="og:image:secure_url" content="' . esc_url( $image_url ) . '" />' . "\n";
+		echo '<meta property="og:image:type" content="image/png" />' . "\n";
+		echo '<meta property="og:image:width" content="1200" />' . "\n";
+		echo '<meta property="og:image:height" content="675" />' . "\n";
+		echo '<meta property="og:image:alt" content="' . esc_attr( $html_title ) . '" />' . "\n";
```

Comment:
What about doing this with an array and a foreach loop? This way it's quite hard to read.

Reply:
Done [here](https://github.com/WordPress/wporg-gp-translation-events/pull/316/commits/41d2c2968990438822edc38a6f48e5e78bc55e57).